docs/Configuration.md

```diff
@@ -195,17 +194,7 @@ English is the Zonemaster default language, but it can be blocked
 from being allowed by RPC-API by not including it in the
 configuration.
 
-In the RPCAPI, `language tag` is used ([Language tag]). The
-`language tags` are generated from the `locale tags`. Each
-`locale tag` will generate two `language tags`, a short tag
-equal to the first two letters (usually the same as a language
-code) and a long tag which is equal to the full `locale tag`.
-If "en_US" is the `locale tag` then "en" and "en_US" are the
-`language tags`.
-
-If there are two `locale tags` that would give the same short
-`language tag` then that is excluded. E.g. "en_US en_UK" will
-only give "en_US" and "en_UK" as `language tags`.
```

Comment:
It looks like this paragraph has been removed with commit 0b0d5af2. I think we should keep somewhere the fact that we remove the short _language tag_ when two (or more) _locale tags_ result in the same short tag (ie: "en_US" and "en_GB" will not give "en").

Reply:
Isn't that clear enough from this?

> * If the *language tag* is a two-character string, it needs to match the
>   first two characters of exactly one *locale tag* in LANGUAGE.locale.
>   (So that it is unambiguous which *locale tag* is matched.)
>
>  E.g. if LANGUAGE.locale is "en_US en_UK sv_SE", all the valid *language tags*
>  are "en_US", "en_UK", "sv_SE" and "sv".

Reply:
From my point of view, both sentences describe the problem from a different perspective. One is starting from two _locale tags_ giving the same two-character _language tag_, the other one starts with one two-character _language tag_ which will match with the beginning of a _locale tag_. 
I think that to understand the output of `get_language_tags`, we should explain that we don't output the two-character _language tag_ when 2 (or more) _locale tags_ are resulting in the same short _language tag_.

Reply:
Sure, I'll make a stab at it. I'll adapt @matsduf's suggestion from above to make it work in its new context.